lib/apis/DeviceSettings.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+
+const _ = require('lodash');
+const debug = require('debug')('voxa');
+const rp = require('request-promise');
+
+class DeviceSettings {
+  constructor(alexaEvent) {
```

Comment:
So, most of this API Classes have a similar format, with a constructor that takes the alexaEvevnt and grabs the apiAccessToken and apiEndpoint, they also have a `getOptions` method, let's go ahead and make an actual base class and share some code